Gemfile.lock

```diff
@@ -691,5 +694,8 @@ DEPENDENCIES
   webmock
   whenever
 
+RUBY VERSION
+   ruby 2.3.0p0
+
```

Comment:
perhaps redo the gemfile.lock since we removed the ruby statement in the gemfile?